crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -413,53 +416,84 @@ pub(crate) struct FromPrettierConfiguration {
     ignore_path: Option<PathBuf>,
 
     /// The translated Biome configuration, from the Prettier configuration
-    biome_configuration: Option<(FormatterConfiguration, JavascriptFormatter)>,
+    formatter_configuration: Option<FormatterConfiguration>,
+
+    /// The translated Biome configuration, from the Prettier configuration
+    javascript_formatter_configuration: Option<JavascriptFormatter>,
 }
 
 impl FromPrettierConfiguration {
     pub(crate) fn store_configuration(
         &mut self,
-        configuration: (FormatterConfiguration, JavascriptFormatter),
+        (formatter, javascript_formatter): (FormatterConfiguration, JavascriptFormatter),
     ) {
-        self.biome_configuration = Some(configuration);
+        self.formatter_configuration = Some(formatter);
+        self.javascript_formatter_configuration = Some(javascript_formatter);
     }
 
-    pub(crate) fn store_configuration_path(&mut self, path: PathBuf) {
-        self.configuration_path = Some(path);
+    pub(crate) fn store_ignored_globs(&mut self, value: StringSet) {
+        let formatter_configuration =
+            self.formatter_configuration
+                .get_or_insert(FormatterConfiguration {
+                    ..NoneState::none()
+                });
+
+        formatter_configuration.ignore = Some(value);
     }
 
-    #[allow(unused)]
-    pub(crate) fn store_ignore_path(&mut self, path: PathBuf) {
-        self.ignore_path = Some(path)
+    pub(crate) fn store_configuration_path(&mut self, path: impl Into<PathBuf>) {
+        self.configuration_path = Some(path.into());
     }
 
-    pub(crate) fn get_biome_configuration(
-        &self,
-    ) -> Option<&(FormatterConfiguration, JavascriptFormatter)> {
-        self.biome_configuration.as_ref()
+    pub(crate) fn store_ignore_path(&mut self, path: impl Into<PathBuf>) {
+        self.ignore_path = Some(path.into())
+    }
+
+    pub(crate) fn as_biome_configuration(&self) -> Configuration {
```

Comment:
I think this method should be called `to_biome_configuration()`. Rust has pretty strict conventions around the `as_`, `to_` and `into_` prefixes: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv